CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main
 
+* **Breaking change**: Upgrade StateComponent to support functional colors. This change requires using https://www.npmjs.com/package/@primer/css-next. 
```

Comment:
I'm not sure how best to communicate this. We need another sentence saying what's next.